### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by Anja1111** [johngodley/redirection#692]
Hi,

Since a few weeks I can't access the redirection plugin anymore from my wp-admin, it says that something has gone wrong, "failed to fetch". It used to work perfectly before.

My REST API works fine as there is a lot of information there :)

I disabled the cache plugins I have. This didn't solve the issue. I can't turn off the other plugins as they are all important, the other plugins I have is Yoast SEO, WP forms lite, WP Content Copy Protection & No Right Click, Social Media and Share Icons (Ultimate Social Media), Really Simple SSL (because I switched to SSL and redirection is required!), Custom Permalinks, Cookie Notice (mandatory by European law) and Akismet Anti-Spam. Disabling that last one didn't work either.

I can't access the plugin anymore to create new redirections and/or manage the existing ones. I would be so happy if this was solved. Below is the info you requested. Thank you for helping!

Anja

```
Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 7.0.26
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://jouwspiegeltje.nl/wp-json/

Error: Failed to fetch
```

**Comment by johngodley** [comment]
Just on the off-chance, do you get the same result if you use another browser?

**Comment by EntranceJew** [comment]
Hello, just dropping by to say that I have this issue as well. @johngodley the issue stems from the fact it's trying to use HTTPS when neither the wordpress address or the site URL specify support of HTTPS. On my server, since HTTPS is not configured for my domain, it gets an nginx error.

**Comment by johngodley** [comment]
@EntranceJew can you show the error output for yours? I think it might be slightly different as the above site does support https

**Comment by EntranceJew** [comment]
@johngodley output appears identical
```
Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 7.0.16-4+deb.sury.org~xenial+1
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://ejew.in/wp-json/

Error: Failed to fetch

Error: Failed to fetch
```

**Comment by Anja1111** [comment]
@johngodley Yay, I just resoled the issue. Now shake and shiver John, you might want to add this option to the common issues ;) 

I found it out because WP couldn't write my .htacess file, so I went on searching as it is writable, but it kept giving issues. I couldn't manage my permalinks, that's why I found out.

On one website I read that the issue appears when your Siteadres is different from your WordPress-adres. So I changed the Siteadres back to https://www instead of https:// (which I prefer, but okay, it's more important that everything is working). 

So the permalink issue with WP was solved and guess what: I can now access your awesome plugin (and a few others) again as well. Yes! I'm happy, so I will close this issue now and I thank you for your suggestion John! 

And yes I also tried other browsers btw. :) This didn't work either. So it's a bit stubborn behavior of Wordpress itself, but okay, it's perfect. Issue solved, happy customer! It wasn't your fault ;)

Have a great day!

Anja
